api/specs/webserver/v0/openapi-auth.yaml

```diff
@@ -0,0 +1,172 @@
+openapi: 3.0.0
+info:
+  title: auth API
+  version: 0.1.0
+  description: authentication and authorization
+servers:
+  - description: API server
+    url: '/v0'
+paths:
+
+  /auth/register:
+    post:
+      operationId: auth_register
+      requestBody:
+        description: user registration
+        content:
+          application/json:
+            schema:
+              $ref: './components/schemas/registration.yaml#/RegistrationType'
+        required: true
+      responses:
+        '200':
+          description: User has been succesfully registered.
+          content:
+            application/json:
+              schema:
+                $ref: './components/schemas/log_message.yaml#/LogMessageEnveloped'
```

Comment:
why response through LogMessage to register? or is it logmessage as in login message?